File: operator-api/src/routes/health.js
import { Hono } from 'hono'
export const healthRouter = new Hono()
const start = Date.now()
healthRouter.get('/health', async (c) => {
  return c.json({ status: 'ok', uptime_sec: Math.round((Date.now() - start) / 1000) })
})

File: server-sigma/internal/routes/health/health.go
package health

import (
	"encoding/json"
	"net/http"
	"time"
)

var start = time.Now()

func Register(mux *http.ServeMux) {
	mux.HandleFunc("/health", func(w http.ResponseWriter, r *http.Request) {
		writeJSON(w, http.StatusOK, map[string]any{
			"status":     "ok",
			"uptime_sec": int(time.Since(start).Seconds()),
		})
	})
}

func writeJSON(w http.ResponseWriter, code int, v any) {
	w.Header().Set("Content-Type", "application/json")
	w.WriteHeader(code)
	_ = json.NewEncoder(w).Encode(v)
}